src/coprocessor/codec/table.rs

```diff
@@ -70,6 +70,13 @@ pub fn check_table_ranges(ranges: &[KeyRange]) -> Result<()> {
     for range in ranges {
         extract_table_prefix(range.get_start())?;
         extract_table_prefix(range.get_end())?;
+        if range.get_start() > range.get_end() {
```

Comment:
Is it valid that `range.get_start() == range.get_end()`?

Reply:
no